src/databricks/sql/thrift_api/TCLIService/ttypes.py

```diff
@@ -36,6 +36,7 @@ class TProtocolVersion(object):
     SPARK_CLI_SERVICE_PROTOCOL_V5 = 42245
     SPARK_CLI_SERVICE_PROTOCOL_V6 = 42246
     SPARK_CLI_SERVICE_PROTOCOL_V7 = 42247
+    SPARK_CLI_SERVICE_PROTOCOL_V8 = 42248
```

Comment:
Please confirm: this regeneration of `ttypes.py` is identical to the original implementation that we reverted, yes? It was necessary so we could capture the new `..._V8` value.

Reply:
Yep